jwst/ramp_fitting/tests/test_ramp_fit.py

```diff
@@ -520,7 +475,6 @@ def test_four_groups_oneCR_orphangroupatend_fit(self, method):
         data = slopes[0]
         np.testing.assert_allclose(data[50, 50], cds_slope, 1e-6)
 
-    # @pytest.mark.skip(reason="not using now")
```

Comment:
So it *is* being used now?

Reply:
This skip, also, has been commented out for a while, so has been in use.

Reply:
Ah, I didn't even notice the `#` at the beginning of the line, thus already rendering it inoperative. Thanks for pointing that out.